bindings/java/src/main/java/org/openucx/jucx/ucp/UcpWorker.java

```diff
@@ -91,15 +91,20 @@ public void setAmRecvHandler(int amId, UcpAmRecvCallback callback) {
         callbackAndWorker[0] = callback;
         callbackAndWorker[1] = this;
         amRecvHandlers.put(amId, callbackAndWorker);
-        setAmRecvHandlerNative(getNativeId(), amId, callbackAndWorker);
+        setAmRecvHandlerNative(getNativeId(), amId, callbackAndWorker, flags);
+    }
+
+    public void setAmRecvHandler(int amId, UcpAmRecvCallback callback) {
+        setAmRecvHandler(amId, callback, UcpConstants.UCP_AM_FLAG_WHOLE_MSG |
+                                         UcpConstants.UCP_AM_FLAG_PERSISTENT_DATA);
```

Comment:
why need to pass UcpConstants.UCP_AM_FLAG_PERSISTENT_DATA?

Reply:
To be the default behavior. If user really don't want it, he can use explicit flag passing

Reply:
why do we want it to be default? it makes it different than C API